api/student/student.controller.js

```diff
@@ -0,0 +1,83 @@
+'use strict'
+
+const StandardError = require('standard-error')
+const axios = require('axios')
+
+class StudentController {
+  constructor (options) {
+    this.options = options || {}
+    this.baseUrl = options.supdataHost
+    this.apiKey = options.supdataApiKey
+    this.authorize = this.authorize.bind(this)
+    this.ping = this.ping.bind(this)
+    this.student = this.student.bind(this)
+    this.consumerMatch = this.consumerMatch.bind(this)
+  }
+
+  authorize (req, res, next) {
+    if (req.authType === 'FranceConnect') {
```

Comment:
"c'est fou !"

Reply:
You shouldn't design something for FranceConnect at this point.

Reply:
🙆‍♂